src/routes.tsx

```diff
@@ -0,0 +1,60 @@
+import ServiceGraphRouteHandler from './pages/ServiceGraph/ServiceGraphRouteHandler';
+import ServiceListPage from './pages/ServiceList/ServiceListPage';
+import IstioConfigPage from './pages/IstioConfigList/IstioConfigListPage';
+import ServiceJaegerPage from './pages/ServiceJaeger/ServiceJaegerPage';
+import ServiceDetailsPage from './pages/ServiceDetails/ServiceDetailsPage';
+import IstioConfigDetailsPage from './pages/IstioConfigDetails/IstioConfigDetailsPage';
+
+const baseName = '/';
```

Comment:
Is this actually used?
If not, please remove. I find this confusing given that we have:
https://github.com/kiali/kiali-ui/blob/83aa48c2d87b60fbc07e486fb81078da76616d85/src/app/History.tsx#L2

Reply:
Yes, I'll remove it, I was looking to fix the error in console with path

Reply:
Maybe move this to the same file to configure all related with routes in the same file

Reply:
Changed